packages/ragbits-cli/src/ragbits/cli/app.py

```diff
@@ -0,0 +1,60 @@
+import json
+from dataclasses import dataclass
+from typing import Any
+
+import typer
+from pydantic import BaseModel
+from rich.console import Console
+from rich.table import Table
+
+
+@dataclass()
+class CliState:
+    """A dataclass describing CLI state"""
+
+    output_type: str = "text"
```

Comment:
It seems to me that `output_type` should be an enum

Reply:
done